nowcasting_dataset/config/model.py

```diff
@@ -98,6 +98,11 @@ class PV(DataSourceMixin):
     )
     pv_image_size_pixels: int = IMAGE_SIZE_PIXELS_FIELD
     pv_meters_per_pixel: int = METERS_PER_PIXEL_FIELD
+    get_center: bool = Field(
```

Comment:
Nice!

Reply:
If you agree, please add a comment to the `description` to say something like "Typically, get_center would be set to true if and only if PVDataSource is used to define the geospatial positions of each example."